brave/brave-core
# fix(wallet): 0x Icon

## Description 

Updates the `0x` icon in the `Swap` screen to one that is visible in `dark-mode`.


Resolves <https://github.com/brave/brave-browser/issues/39551>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Set your browser to dark mode in brave://settings.
2. Open the `Wallet` and navigate to the `Swap` screen.
3. Open the `Choose quote provider` modal.
4. You will see the `0x` is visible in `dark-mode`

Before:

![Screenshot 4](https://github.com/brave/brave-browser/assets/40611140/2bdfd11e-7a95-4b2a-9ab7-5c8105681c7d)

![Screenshot 5](https://github.com/brave/brave-browser/assets/40611140/9973c2bc-c882-4d19-970c-3ca9832a14b8)

After:

![Screenshot 7](https://github.com/brave/brave-core/assets/40611140/5331a973-9005-4d48-aca8-1a2a6e3e40f2)

![Screenshot 8](https://github.com/brave/brave-core/assets/40611140/520450f1-bc6b-4468-885d-712ac1446d30)
